Handler/Plan.hs

```diff
@@ -0,0 +1,22 @@
+module Handler.Plan where
+
+import Import
+
+getPlansR :: Handler Html
+getPlansR = do
+    plans <- runDB $ selectList [] [Asc PlanSort]
+    Entity _ user <- requireAuth
```

Comment:
This is not calling the `Entity` constructor. It's pattern matching on `Entity`. Since we don't care about the `UserId` field, we ignore it with a wildcard pattern (`_`).